shared/message/platform/webots/messages.proto

```diff
@@ -83,20 +85,30 @@ message Message {
     string      text         = 2;
 }
 
+/// NUbots specific data sent from our own controllers rather than the official RoboCup Webots controller
+message OdometryGroundTruth {
+    /// Indicates if this message exists
+    bool exists = 1;
```

Comment:
Use protobuf's optional rather than a flag.

Reply:
Optional doesn't exist anymore, and I don't think that Neutron respects it regardless